parm/postcntrl_gefs_aerosol.xml

```diff
@@ -10,64 +10,997 @@
     <sigreftime>fcst</sigreftime>
     <prod_status>oper</prod_status>
     <data_type>fcst</data_type>
-    <gen_proc_type>fcst</gen_proc_type>
+    <gen_proc_type>ens_fcst</gen_proc_type>
```

Comment:
@zhanglikate The "ens_fcst" is not an official type of generating process at 
https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-3.shtml
Please revert it.